Formula/pastebinit.rb

```diff
@@ -20,7 +19,7 @@ def install
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
     bin.install "pastebinit"
-    etc.install "pastebin.d"
+    cp_r "pastebin.d", etc
```

Comment:
Something went weird when I did the pull: even though there's only 1 commit in this PR, and under its "Files changed" it shows the `etc.install` being there, there are now 2 `pastebinit 1.5` commits in `master`'s history, and one of them changes to `cp_r`.

```
$ git log --oneline | head
8b3e3f4 pastebinit: update 1.5 bottle.
6d311f6 pastebinit 1.5
9e3e897 pastebinit 1.5
29c84fe gpx: add 2.4.3 bottle.
1113d62 gpx 2.4.3 (new formula)
df02606 lbdb: update 0.41 bottle.
```

I'll fix it in a new commit shortly.

That shouldn't require a bottle rebuild, because the bottle's installation of `etc/` files is always done the same way, regardless of how those files were installed during the from-source installation, right?